atrium-xrpc-client/Cargo.toml

```diff
@@ -20,20 +20,27 @@ reqwest = { workspace = true, optional = true }
 surf = { workspace = true, optional = true }
 
 [features]
-default = ["reqwest-native"]
+default = ["reqwest-default-tls"]
 isahc = ["dep:isahc"]
-reqwest-native = ["reqwest/native-tls"]
-reqwest-rustls = ["reqwest/rustls-tls"]
+reqwest = ["dep:reqwest"]
+reqwest-default-tls = ["reqwest/default-tls"]
 surf = ["dep:surf"]
 
-[dev-dependencies]
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+reqwest.workspace = true
```

Comment:
This makes `reqwest` non-optional on `wasm32`, which is a little odd but I guess makes sense given that `atrium-xrpc-client` is completely empty when no features are enabled. If a non-`reqwest` backend gains `wasm32` support then changing this to be default-enabled instead at that time is fine.